python/cudf/dataframe/dataframe.py

```diff
@@ -284,6 +284,14 @@ def __len__(self):
         """
         return self._size
 
+    def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
+        if method == '__call__':
+            if 'sqrt' in str(ufunc):
```

Comment:
If the method isn't `sqrt` we'll return None here which will be super unexpected. We should have an else clause with a `NotImplementedError` if the ufunc isn't sqrt.